### PR TITLE
Graph Plotter App

### LINKED ISSUE #281: Graph Plotter app
## 🚀 Feature

Graph plotter app made in flutter which will plot the graph by just entering the x and y-axis values.

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes I have read it.

## Motivation

Plotting some data in graph can give overall picture of data flow hence this app idea came to my mind.

## Pitch

It can be used to check the overall marks of a student to analyze the overall result of him/her whether he/she is in school or college.
This app can also help to plot the expenses graph per year or per month or per week to analyze the money outflow from their side and they can change their money outflow according to their needs.

### PR DESCRIPTION
**Pull request by Rahul2570089**
GSSOC'22

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

- Yes , I have gone through it.

### Description

Fixes #281 Graph plotter app in flutter

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

### CODE REVIEW

**Review by photon149 — COMMENTED** [review]
@Rahul2570089 Great Work 👍🏻 
Just Add a section in your README regarding steps to run your app.
Everything else looks good to me.

**Comment by Rahul2570089** [comment]
I have already added the steps in my app's readme file. Do you mean to add in the readme file in "Mobile Applications" folder ?

**Comment by photon149** [comment]
@Rahul2570089 No it's fine.
@Kushal997-das @prakhar3062 have a look once.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Rahul2570089 Please delete all unwanted files only add those files those are really requires.

**Comment by Rahul2570089** [comment]
@Kushal997-das Sorry but can you please specify which files because according to me I have removed all files which are not required.

**Comment by Kushal997-das** [comment]
@Rahul2570089 Just add your program files and using those you can run this project add them only and remaining files delete them , like your dot(.) files.

**Comment by Rahul2570089** [comment]
Okay

**Comment by Rahul2570089** [comment]
I have updated the pull request. Can you please check?

**Comment by Kushal997-das** [comment]
> @Rahul2570089 Just add your program files and using those you can run this project add them only and remaining files delete them , like your dot(.) files.

??????

**Comment by Rahul2570089** [comment]
I have removed the dot(.) files

**Comment by Rahul2570089** [comment]
Untitled

File: Mobile Applications/Intermediate/graphplotter/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Graph Plotter App
+
```
Comment (Kushal997-das) [inline Mobile Applications/Intermediate/graphplotter/README.md:2]: Can you please add here some words about your project .

Comment (Kushal997-das) [reply Mobile Applications/Intermediate/graphplotter/README.md:2]: @Rahul2570089

Comment (Rahul2570089) [reply Mobile Applications/Intermediate/graphplotter/README.md:2]: Okay

**Review by Kushal997-das — APPROVED** [review]

**Comment by Rahul2570089** [comment]
Thank you ❤️

**Pull request merged**